debian/control

```diff
@@ -2,7 +2,7 @@ Source: debathena-pyhesiodfs
 Section: debathena/net
 Priority: extra
 Maintainer: Debathena Project <debathena@mit.edu>
-Build-Depends: cdbs, debhelper, config-package-dev, python-support, python (>= 2.6~)
+Build-Depends: cdbs, debhelper, dh-systemd (>= 1.5), config-package-dev, python-support, python (>= 2.6~)
```

Comment:
Since this patch is really intended for "future" (post-jessie and post-vivid) releases of Debian and Ubuntu, does this mean that we have to create a new `debian` branch for new releases only?